util/util.c

```diff
@@ -45,3 +46,9 @@ unsigned int get_random(void)
 
 	return rand_r(&seed);
 }
+
+bool check_env(const char *var)
+{
+	const char *env_value = getenv(var);
```

Comment:
Please add extra line here to follow kernel coding style.

Besides that, patch is ok.

Thanks